.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -12,18 +12,13 @@ body:
       label: 📝 Preliminary Checks
       description: Please read these carefully.
       options:
-        - label: I checked that all ports are open and not blocked by my ISP / hosting provider.
-          required: true
-        - label: I know that SSL errors are likely the result of a wrong setup on the user side and not caused by DMS itself. I'm confident my setup is correct.
+        - label: The [documentation on debugging](https://docker-mailserver.github.io/docker-mailserver/latest/config/debugging/), tried the proposed steps to debug the problem, but was still unable to resolve the issue.
```

Comment:
This technically makes the next item redundant, depends if you still want them to explicitly see and confirm the expectation of searching previous issues (_we still get users that don't_).